fix up build

dont strip the binary as portage will take care of it

--- svgalib-1.9.25/Makefile.cfg
+++ svgalib-1.9.25/Makefile.cfg
@@ -44,7 +44,7 @@ CONFDIR = $(SRCDIR)/src/config
 # Common prefix for installation directories.
 # NOTE: This directory must exist when you start the install.
 TOPDIR=
-prefix = $(TOPDIR)/usr/local
+prefix = $(TOPDIR)/usr
 exec_prefix = $(prefix)
 
 # Directory where the shared stubs and static library will be installed.
@@ -313,14 +313,13 @@
 WARN     = -Wall -Wstrict-prototypes
 INCLUDES = -I$(srcdir)/include -I.
 CFLAGS   = $(WARN) $(DLLFLAGS) $(INCLUDES) $(OPTIMIZE) $(DEFINES) 
-LDFLAGS   = -s
 
 # additional flags for shared lib.
 DLLFLAGS = -fPIC -DPIC
 
 # Utilites used.
 AR	  = ar
-INSTALL_PROGRAM	= install -sm755
+INSTALL_PROGRAM	= install -m755
 INSTALL_SCRIPT	= install -m755
 INSTALL_SHLIB	= install -m755
 INSTALL_DATA	= install -m644
--- svgalib-1.9.25/Makefile
+++ svgalib-1.9.25/Makefile
@@ -99,8 +99,6 @@
 		(cd $(sharedlibdir); \
 		 ln -sf $$foo `echo $$foo | sed 's/\.so\..*/.so/'` ); \
 	done
-	@./fixldsoconf
-	-ldconfig
 
 installstaticlib: static
 	@echo Installing static libraries in $(libdir).
--- svgalib-1.9.25/gl/Makefile
+++ svgalib-1.9.25/gl/Makefile
@@ -29,7 +29,7 @@ all:	libvgagl.a
 .PHONY: all clean dep
 
 libvgagl.so.$(VERSION): $(MODULES)
-	$(CC) -s -shared -Wl,-soname,libvgagl.so.$(MAJOR_VER) -o libvgagl.so.$(VERSION) \
+	$(CC) -shared -Wl,-soname,libvgagl.so.$(MAJOR_VER) -o libvgagl.so.$(VERSION) \
 	  $(MODULES)
 
 libvgagl.a: $(MODULES)
--- svgalib-1.9.25/threeDKit/Makefile
+++ svgalib-1.9.25/threeDKit/Makefile
@@ -58,7 +58,6 @@
 $(sharedlibdir)/lib3dkit.so.$(VERSION): lib3dkit.so.$(VERSION)
 	$(INSTALL_SHLIB) $< $(sharedlibdir)/$<
 	(cd $(sharedlibdir); ln -sf lib3dkit.so.$(VERSION) `echo lib3dkit.so.$(VERSION) | sed 's/\.so\..*/.so/'` )
-	-ldconfig
 
 install: $(sharedlibdir)/lib3dkit.so.$(VERSION) installheaders
 
--- svgalib-1.9.25/src/Makefile
+++ svgalib-1.9.25/src/Makefile
@@ -453,7 +453,6 @@
 
 $(sharedlibdir)/libvga.so.$(VERSION): libvga.so.$(VERSION)
 	$(INSTALL_SHLIB) $< $(sharedlibdir)/$<
-	-ldconfig
 
 install: $(sharedlibdir)/libvga.so.$(VERSION)
 
